src/components/Modals/GroundProperties.vue

```diff
@@ -0,0 +1,97 @@
+<!-- OpenStudio(R), Copyright (c) 2008-2016, Alliance for Sustainable Energy, LLC. All rights reserved.
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+(1) Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+(2) Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+(3) Neither the name of the copyright holder nor the names of any contributors may be used to endorse or promote products derived from this software without specific prior written permission from the respective party.
+(4) Other than as required in clauses (1) and (2), distributions in any form of modifications or other derivative works may not use the "OpenStudio" trademark, "OS", "os", or any other confusingly similar designation without specific prior written permission from Alliance for Sustainable Energy, LLC.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER, THE UNITED STATES GOVERNMENT, OR ANY CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. -->
+
+<template>
+  <ModalBase
+    class="ground-props-modal"
+    title="Ground Properties"
+    @close="$emit('close')"
+  >
+    <div class="ground-properties">
+      <p><label class="input-text">Floor Offset <input type="text" v-model="floor_offset" /></label></p>
+      <p><label class="input-text">Asimuth Angle <input type="text" v-model="azimuth_angle" /></label></p>
+      <p><label class="input-text">Tilt Angle <input type="text" v-model="tilt_angle" /></label></p>
+      <p><label class="input-text">x-shift <input type="text" v-model="x_shift" /></label></p>
+      <p><label class="input-text">y-shift <input type="text" v-model="y_shift" /></label></p>
+    </div>
+  </ModalBase>
+</template>
+
+<script>
+import ModalBase from './ModalBase.vue';
```

Comment:
Why do we put this up here instead of in the main script tag for the vue file?

Reply:
ah, there was a terrible copy-paste accident, and this was the damage.